edx_proctoring/api.py

```diff
@@ -818,6 +818,7 @@ def get_exam_attempt_data(exam_id, attempt_id, is_learning_mfe=False):
         'critically_low_threshold_sec': critically_low_threshold,
         'course_id': exam['course_id'],
         'attempt_id': attempt['id'],
+        'external_id': attempt['external_id'],
```

Comment:
test_get_exam_attempt_data doesn't test all fields but since this new field would cause a hard to understand error if it breaks I think it's worth adding a check

Reply:
yeah that's fair. I didn't want to get into the business of testing all of the fields for this small change but a one-off couldn't hurt.